atlp-rwanda/cabal-phantom-backend
# # usCaESGS-internationalisation-of-phantom

**What does this PR do?**
    -This PR is about translating welcome message into 3 languages
**Description of Task to be completed?**
    -this PR enable user to select language of his/her choice (english, french and kinyarwanda) and see contents in his/her choice
**How should this be manually tested?**
   1. Clone this Repo if you don't have
   2. type git checkout `origin/ft_backend_internationalise_qSwfwWJp`
  3. start server by writing `npm run dev`
  4. Open your postman and type in `GET:localhost:5000` by Default messages will displayed in english
     . then go to Header type in key field **Accept-Language** and type **2 first characters of language you choose** (ex:ki)
     . then click on Send to see what happen
 5.Also you can open your browser and type in URL **localhost:5000/?lang=fr**
     . you can change fr to ki or en

**Any background context you want to provide?**
    * en: English
    * fr: French
    * ki: kinyarwanda

**What are the relevant pivotal tracker stories?**
    [usCaESGS](https://trello.com/c/usCaESGS/20-support-multiple-languages)

**Screenshots (if appropriate)**
   <img width="1440" alt="Screen Shot 2020-09-28 at 15 47 50" src="https://user-images.githubusercontent.com/41394162/94440507-0c711780-01a2-11eb-9976-6ac682db0b67.png">
